### ISSUE TITLE
A11y: The alphabetical shortcut navigation inside the keyboard shortcut windo doesn't work

### ISSUE THREAD

**Issue by akash07k** [microsoft/vscode#96607]
Issue description:
The alphabetical shortcut navigation inside the keyboard shortcut windo doesn't work. Due to this, we can't navigate to any keyboard shortcut command by pressing the corresponding letter/alphabet key, let's say, "Open the editor": "O", so, since "open editor" starts with "O", we should be able to navigate to it by pressing "O" key on our keyboard.

Expected:
Whenever we press an alphabet key on the keyboard, we should navigate to the corresponding keybinding shortcut. For example, if we press "O", we should navigate to the keybindings starting from "O". (We can take the example of windows explorer for this).

- VSCode Version:
- OS Version:

Steps to Reproduce:

1. Open Visual Studio Code/ Visual Studio Code Insider.
2. Go to File menu>Preferences>Keyboard shortcuts. (The shortcut key for the keyboard shortcuts can also be used).
3. Now navigate inside the keyboard shortcut window and press any corresponding alphabet to go straight to any shortcut key.
4. For example, press "O" to go to the commands starting from "O".
5. Observe, nothing happens.

Does this issue occur when all extensions are disabled?: Yes/No
Yes.

**Comment by isidorn** [comment]
@sandy081 seems like the Keybindings Editor list does not provide a Navigation Provider for the list. This should be an easy fix, thus assigning to May. Let me know what you think?

**Comment by isidorn** [comment]
Pushed a fix which seems to work fine for me.
@sandy081 let me know if you like it or if I should change something.

**Comment by akash07k** [comment]
@isidorn Thanks.
When can I test whether it is fixed?

**Comment by isidorn** [comment]
@akash07k a new insiders with this fix should be out on Friday. Thanks!

**Comment by akash07k** [comment]
Ok @isidorn, thanks a lot for your efforts

**Comment by sandy081** [comment]
@isidorn Thanks

**Comment by isidorn** [comment]
To verify:
1. Open keybindings shortcuts view
2. Focus on the list
3. Start typing - we should navigate as you type
